### PR TITLE
Drone config

### PR DESCRIPTION
**Pull request by mmj-ssb**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -37,9 +45,20 @@ steps:
       webhook:
         from_secret: slack_webhook_url
 #      channel: bip-ci-test -- Choosing channel is not enabled yet
+      template: >
+        {{#success build.status}}
+          *Success*! {{uppercasefirst build.event}} - {{build.link}}. Good job *{{build.author}}*!
+        {{else}}
+          *Failure*! {{uppercasefirst build.event}} - {{build.link}}.
```
Comment (takvamborgen) [inline .drone.yml]: You forgot `Awful job *{{build.author}}*!`

File: .drone.yml [hunk 2]
```diff
@@ -37,9 +45,20 @@ steps:
...
+        {{/success}}
     when:
       status: [ success, failure ]
 
+trigger:
+  ref:
+  - refs/heads/master
+  - refs/pull/*/head
+
```
Comment (takvamborgen) [inline .drone.yml]: Not `develop`?

File: .drone.yml (hunk 1 above)
Comment (mmj-ssb) [reply .drone.yml]: Do we really want to place blames in Slack? ;)

File: .drone.yml (hunk 2 above)
Comment (mmj-ssb) [reply .drone.yml]: Havent been using a develop branch in this app yet. Can change to that if it makes sense later.

File: .drone.yml (hunk 1 above)
Comment (takvamborgen) [reply .drone.yml]: Always place the blame :)

**Pull request merged**